cloudinit/util.py

```diff
@@ -555,6 +555,10 @@ def is_FreeBSD():
     return system_info()['variant'] == "freebsd"
 
 
+def is_NetBSD():
```

Comment:
please add `@lru_cache()` to this function

Reply:
Good idea.